### ISSUE TITLE
NTR: alpha-L-iduronidase enzyme low or absent (Lab: blood chemistry: enzymes: leukocyte)

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#8131]
New term request alpha-L-iduronidase enzyme low or absent (c0020818):

**Comment by pnrobinson** [comment]
@MickeySegal

Are these terms are synonyms? What is the context of your term?
[Decreased iduronate sulfatase level HP:0034203](https://hpo.jax.org/browse/term/HP:0034203)

**Comment by MickeySegal** [comment]
That is in Hunter syndrome and multiple sulfatase deficiency.  

This is in https://omim.org/entry/607014

**Comment by pnrobinson** [comment]
@MickeySegal 

Adding new term: 
- ID: HP:6000910
- label: Diminished tissue alpha-L-iduronidase activity
- parents: [Abnormal tissue enzyme concentration or activity (HP:6000231)](https://hpo.jax.org/app/browse/term/HP:6000231)
- pmid: PMID:818611
- def: Concentration or activity of alpha-L-iduronidase (IDUA; EC 3.2.1.76) below the lower limit of normal. FECH enzyme can be measured in multiple tissues including cultured fibroblasts and leukocytes.
- comment: 
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/8131
- synonyms:
